src/view/components/formikReactSpectrum3/radioGroupWithDataElement.jsx

```diff
@@ -75,14 +75,17 @@ const RadioGroupWithDataElement = ({
   });
```

Comment:
The setConsent tests found some bugs in the radioGroupWithDataElement component.